app/serializers/reviewable_score_serializer.rb

```diff
@@ -42,7 +42,39 @@ def reason
 
     if link_text
       link = build_link_for(object.reason, link_text)
-      text = I18n.t("reviewables.reasons.#{object.reason}", link: link, default: object.reason)
+
+      if object.reason == "watched_word"
+        if object.context.nil?
```

Comment:
Would probably extract this whole block into some `watched_words_link` method to keep `reason` short